packages/smarthr-ui/src/components/ResponseMessage/ResponseMessage.tsx

```diff
@@ -25,22 +25,17 @@ const responseMessage = tv({
   },
 })
 
+const ICON_MAPPER = {
+  info: FaCircleInfoIcon,
+  success: FaCircleCheckIcon,
+  warning: WarningIcon,
+  error: FaCircleExclamationIcon,
+  sync: FaRotateIcon,
+}
```

Comment:
あったほうが安心な気はする、ぐらいの提案です〜

```suggestion
} as const
```